### ISSUE TITLE
INSTALL_PATH was dropped from Makefile in 075a35a6d390167b77b687e067dd0ba593e7f624

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/leveldb#111]
```
What steps will reproduce the problem?
1. Make leveldb on OS X
2. Note lack of absolute install path in `-install_name`
3. Attempt to build against leveldb dylib on OS X and fail

What is the expected output? What do you see instead?

g++  -dynamiclib -install_name libleveldb.dylib.1 

instead of

g++  -dynamiclib -install_name /path/to/current/directory/libleveldb.dylib.1 

What version of the product are you using? On what operating system?

Anything after SHA 075a35a6d390167b77b687e067dd0ba593e7f624

Please provide any additional information below.

It looks like when

$(SHARED3):
    $(CXX) $(LDFLAGS) $(PLATFORM_SHARED_LDFLAGS)$(SHARED2) $(CXXFLAGS) $(PLATFORM_SHARED_CFLAGS) $(SOURCES) -o $(SHARED3)

was made, the $(INSTALL_PATH) param was dropped completely.

The attached patch corrects this.
```

---

Original issue reported on code.google.com by `j...@somethingsimilar.com` on 21 Aug 2012 at 5:44

Attachments:
- [install_path.patch](https://storage.googleapis.com/google-code-attachments/leveldb/issue-111/comment-0/install_path.patch)

**Comment by GoogleCodeExporter** [comment]
```
Issue 112 has been merged into this issue.
```

---

Original comment by `dgrogan@chromium.org` on 22 Aug 2012 at 6:08

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.7.0.

```

---

Original comment by `san...@google.com` on 16 Oct 2012 at 11:29
- Changed state: **Fixed**
